src/main/java/edu/nextstep/racingcarwinner/domain/Car.java

```diff
@@ -11,46 +11,46 @@
 import java.util.Objects;
 
 public class Car {
-    private final String name;
-    private int distance;
+    private final Name name;
+    private Distance distance;
 
     public Car(String name) {
         this(name, 0);
     }
 
     public Car(String name, int distance) {
-        this.name = name;
-        this.distance = distance;
+        this.name = new Name(name);
+        this.distance = new Distance(distance);
     }
 
     public void move(RacingStrategy racingStrategy) {
         if (racingStrategy.movable()) {
-            this.distance++;
+            this.distance = distance.move();
         }
     }
 
+    public String getName() {
+        return name.getName();
+    }
+
+    public int getDistance() {
+        return distance.getDistance();
+    }
+
```

Comment:
```suggestion
    public String getName() {
        return name.value();
    }

    public int getDistance() {
        return distance.value();
    }

```
원시값 포장 객체의 경우 이렇게 `value()` 메소드를 통해서 값을 반환하도록 하면 어떨까요?